### ISSUE TITLE
Git authentication issue introduced on version 1.42.0

### ISSUE THREAD

**Issue by norelina** [microsoft/vscode-remote-release#2340]
- VSCode Version: 1.42.0 (also tested on 1.43.0-insider. Bug NOT present on 1.41.0)
- Local OS Version: Windows 10 64-bit
- Remote OS Version: Linux - Alpine 3.10
- Remote Extension/Connection Type: Docker

Steps to Reproduce:

1. Install Visual Studio Code - version 1.42.0
2. Install Remote - Containers extension
3. View>Command Palette>Remote - Containers: Open Folder in Container
4. Use Alpine 3.10 (Simple Alpine 3.10 container with Git installed)
5. Open Terminal: git pull (repo cloned using HTTPS)

**Actual Results:** The following error is displayed and the user is prompted to enter password.
```
f() { node /tmp/vscode-remote-containers-777b*****************7ee5.js $*; }; f get: line 1: node: not found. 
```
Node is not installed on the image (and shouldn't have to be)

**Expected Results:** To successfully Pull

**Manual workaround (determined by looking at version 1.41.0):**
In .gitconfig, replace:
```
[credential]
        helper = "!f() { node /tmp/vscode-remote-containers-777b*****************7ee5.js $*; }; f"
```
with
```
[credential]
	helper = "!f() { command -v code >/dev/null 2>&1 && code --gitCredential $*; }; f"
```

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Comment by jesseryoungVUMC** [comment]
Seeing this exact same issue . My current work-around has been:
```
ln -s "$(ls ~/.vscode-server/bin/* -d)/node" /usr/local/bin
```

**Comment by chrmarti** [comment]
Fixed in version 0.101.1.
